### PR TITLE
Link Twitter account.

### PR DESCRIPTION
**Pull request by johnjohndoe**

### CODE REVIEW

**Comment by ligi** [comment]
Unsure about this - should we promote twitter?

**Comment by vog** [comment]
I think it is okay, as it is actively used.

However, we already use the `@` sign as location indicator (e.g. `@cbase`), so it is confusing to use it also for the twitter link. Why not doing it the same way as with Meetup, and call the link simply "Twitter"?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Berlin Hack and Tell
 
-Last Tuesday on each(~) month | [Meetup Page](https://www.meetup.com/de-DE/Berlin-Hack-and-Tell) | [@c-base](https://www.c-base.org)
+Last Tuesday on each(~) month | [Meetup Page](https://www.meetup.com/de-DE/Berlin-Hack-and-Tell) | [@BerlinHacknTell](https://twitter.com/BerlinHacknTell) | [@c-base](https://www.c-base.org)
```
Comment (vog) [inline README.md]: Proposal: Name the link "Twitter".

```
[Twitter](https://twitter.com/BerlinHacknTell)
```

**Comment by vog** [comment]
@stefanhoth What do you think? To link or not to link to Twitter?

File: README.md (hunk 1 above)
Comment (stefanhoth) [inline README.md]: I agree, probably clearer with the Twitter label

**Comment by vog** [comment]
Adjusted link and accepted pull request.

**Pull request merged**